front/app/containers/ProjectsShowPage/timeline/setPhaseURL.ts

```diff
@@ -0,0 +1,26 @@
+import { IPhaseData } from 'services/phases';
+import { IProjectData } from 'services/projects';
+import { Locale } from 'typings';
+import { isNilOrError } from 'utils/helperUtils';
+
+export default function setPhaseUrl(
+  phase: IPhaseData | null,
```

Comment:
Little bit of nitpicking on naming: I'd call `phase` here `selectedPhase`.

I would also be stricter on what you pass through. No errors, null, undefined. Only call when the data is correct. And perhaps only pass the selectedPhaseId, since we don't need more from the phase.